cmd/applyconfig/applyconfig.go

```diff
@@ -316,9 +317,11 @@ func main() {
 			return apply(o.kubeConfig, o.context, path, o.user.val, !o.confirm)
 		}
 
-		adminErr = applyConfig(o.directory, "admin", f)
-		if adminErr != nil {
-			logrus.Errorf("There were failures while applying admin config")
+		for _, dir := range o.directories.Strings() {
+			if err := applyConfig(dir, "admin", f); err != nil {
+				adminErrs = append(adminErrs, err)
+				logrus.WithError(err).Error("There were failures while applying admin config")
```

Comment:
Do the errors returned by `applyConfig` have enough context to determine which file/directory? Would including `dir` in the log be helpful?

Reply:
Yeah, when we walk the dir we log each folder and we also log the `oc` command, so I think that should be enough